codec-stomp/src/main/java/io/netty/handler/codec/stomp/StompSubframeDecoder.java

```diff
@@ -59,6 +60,7 @@
 
     private static final int DEFAULT_CHUNK_SIZE = 8132;
     private static final int DEFAULT_MAX_LINE_LENGTH = 1024;
+    private static final Pattern HEADER_DELIMITER_PATTERN = Pattern.compile(":");
```

Comment:
For splitting with one-char String better use `String#split` instead of `Pattern`, [there is an optimization for this](http://hg.openjdk.java.net/jdk8/jdk8/jdk/file/687fd7c7986d/src/share/classes/java/lang/String.java#l2316).

Reply:
@fenik17 Thanks, I agree it is better approach for this case.